libslide/src/utils/grammar.rs

```diff
@@ -48,19 +48,19 @@ fn get_flattened_binary_args(expr: Rc<Expr>, op: BinaryOperator) -> VecDeque<Rc<
             {
                 args.push_front(arg);
             }
-            args.append(&mut get_flattened_binary_args(Rc::clone(&child.rhs), op));
-            args
+            args.extend(&mut get_flattened_binary_args(Rc::clone(&child.rhs), op).drain(..));
+            args.into_iter().collect()
         }
```

Comment:
out of curiosity what does .. op mean? It says "and the rest" on the docs which confused me

Reply:
`drain` takes a range to drain. `..` is a range to the entire container (eq to infinite range)